docs/docsource/reference/prometheus/index.rst

```diff
@@ -0,0 +1,146 @@
+Prometheus
+==========
+
+One of several dashboards available to Zenko users, Prometheus provides insight
+into Kubernetes functionalities not accessible through the Backbeat API. 
```

Comment:
I would rephrase to:

> One of several dashboards available to Zenko users, Prometheus provides insight into Kubernetes platform (metalk8s) and Zenko functionality.

Because Prometheus scope is larger than what Backbeat API is meant to expose, as it covers Cloudserver, MongoDB or other platform-wide components not related to backbeat, I don't think it's worth mentioning Backbeat API here.

What is mentioned below in "Exposure through the Backbeat API is under development" should be enough I think.

Reply:
done